Remove a bunch more indirection.

mikepurvis opened the pull request:
Mainly affects the inertia properties.

@Shokoofeh to review.

mikepurvis committed:
Remove a bunch more indirection.

Mainly affects the inertia properties.

mikepurvis commented:
Shouldn't be any functional change here. I did a diff on the xacro processed output, and it's identical apart from comment changes.

shokoofeh commented on grizzly_description/urdf/base.urdf.xacro:
```diff
@@ -172,20 +128,10 @@ This program contains the description of the robot base model for the Clearpath
+  <grizzly_wheel fr="rear" lr="left" flipY="0" parent="chassis_link"
+                 translateX="-1" translateY="1" />
+  <grizzly_wheel fr="rear" lr="right" flipY="1" parent="chassis_link"
+                 translateX="-1" translateY="-1" />
```
Only for more readability, I suggest to follow the same order that you used for front wheels: fr  lr  parent  translateX  translateY  flipY

shokoofeh commented:
I test it in simulation. Everything if fine. LGTM

mikepurvis committed:
Merge branch 'indigo-devel' of https://github.com/g/grizzly into remove-properties

mikepurvis committed:
A few more tidy-ups here.

- More joints to after their respective child links.
- Group wheels together at the end.
- Re-add IMU orientation.

mikepurvis commented on grizzly_description/urdf/base.urdf.xacro:
```diff
@@ -147,16 +103,16 @@ This program contains the description of the robot base model for the Clearpath
   <joint name="imu_joint" type="fixed">
     <origin xyz="-${chassis_length/2} 0 0"
-            rpy="0 0 0" />
+            rpy="${M_PI} ${M_PI/2} 0" />
```
This orientation should have remained, as the IMU in the user bay is not mounted level with the robot— sorry for the confusion here. This shouldn't impact robot_localization, I don't think, should it?

shokoofeh replied:
This orientation does impact robot_localization. 
As we discussed, the orientation will be set as it is mount in the user bay but in the imu controller plugin for Gazebo the frame id will be set as base_link.
